site/blog/2022-10-28-optimize-s3-endpoints.mdx

```diff
@@ -0,0 +1,126 @@
+---
+slug: optimize-your-s3-endpoints
+title: Optimize your S3 endpoints for cost and efficiency
+authors: [yrobla]
+---
+
+# Optimize your S3 endpoints for cost and efficiency
+
+Are you using S3 buckets as part of your cloud deployments? How are you accessing them?
+
+When not configured properly, access to the S3 buckets may need internet access. One simple but costly way
+is to rely on <a href="https://docs.aws.amazon.com/en_en/vpc/latest/userguide/vpc-nat-gateway.html" target="_blank">NAT gateways</a>.
+
+But there is a more optimal solution, that is creating <a
```

Comment:
Formally you're not supposed to begin a sentence with words like "but" or "and", etc. But that's also English teachers imposing Romance language grammar rules on Germanic English, so we can ignore that. ;) (Some people do get annoyed by that, though.)

Reply:
spanglish in use :)